cppcache/src/TcpSslConn.hpp

```diff
@@ -67,14 +81,18 @@ class TcpSslConn : public TcpConn {
              const char* privkeyfile, const char* pemPassword)
       : TcpConn(ipaddr, waitSeconds, maxBuffSizePool),
         m_ssl(nullptr),
+        m_sniPort(0),
+        m_sniHostname(""),
```

Comment:
Initializing a `std::string` with `""` is redundant.

Reply:
Fixed